ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EjbJarParsingDeploymentUnitProcessor.java

```diff
@@ -309,6 +310,7 @@ static Map<String, AbstractMetaDataParser<?>> createJbossEjbJarParsers() {
         parsers.put(EJBBoundMdbDeliveryMetaDataParser11.NAMESPACE_URI_1_1, EJBBoundMdbDeliveryMetaDataParser11.INSTANCE);
         parsers.put(EJBBoundMdbDeliveryMetaDataParser12.NAMESPACE_URI_1_2, EJBBoundMdbDeliveryMetaDataParser12.INSTANCE);
         parsers.put(EJBBoundMdbDeliveryMetaDataParser20.NAMESPACE_URI_2_0, EJBBoundMdbDeliveryMetaDataParser20.INSTANCE);
+        parsers.put(EJBBoundMdbDeliveryMetaDataParser30.NAMESPACE_URI_3_0, EJBBoundMdbDeliveryMetaDataParser30.INSTANCE);
```

Comment:
With an enumerated schema and parameterized parser, you would not need to touch this file for subsequent versions.
e.g.
https://github.com/wildfly/wildfly/blob/main/ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EjbJarParsingDeploymentUnitProcessor.java#L330